website/docs/backends/types/gcs.html.md

```diff
@@ -59,19 +85,22 @@ The following configuration options are supported:
     (Optional) Local path to Google Cloud Platform account credentials in JSON
     format.  If unset, [Google Application Default
     Credentials](https://developers.google.com/identity/protocols/application-default-credentials)
-    are used.  The provided credentials need to have the
-    `devstorage.read_write` scope and `WRITER` permissions on the bucket.
+    are used.  The provided credentials must have Storage Object Admin role on the bucket.
     **Warning**: if using the Google Cloud Platform provider as well, it will
     also pick up the `GOOGLE_CREDENTIALS` environment variable.
+ * `impersonate_service_account` - (Optional) The service account to impersonate for accessing the State Bucket.
+    You must have `roles/iam.serviceAccountTokenCreator` role on that account for the impersonation to succeed. 
+    If you are using a delegation chain, you can specify that using the `impersonate_service_account_delegates` field.
+    Alternatively, this can be specified using the `GOOGLE_IMPERSONATE_SERVICE_ACCOUNT` environment
+    variable.
+ * `impersonate_service_account_delegates` - (Optional) The delegation chain for an impersonating a service account as described [here](https://cloud.google.com/iam/docs/creating-short-lived-service-account-credentials#sa-credentials-delegated).
  * `access_token` - (Optional) A temporary [OAuth 2.0 access token] obtained
    from the Google Authorization server, i.e. the `Authorization: Bearer` token
    used to authenticate HTTP requests to GCP APIs. This is an alternative to
    `credentials`. If both are specified, `access_token` will be used over the
    `credentials` field.
  *  `prefix` - (Optional) GCS prefix inside the bucket. Named states for
     workspaces are stored in an object called `<prefix>/<name>.tfstate`.
- *  `path` - (Deprecated) GCS path to the state file of the default state. For
```

Comment:
Deprecated values should probably still be documented until their removal, right?

Reply:
It was deprecated more than 3 years ago.

https://github.com/hashicorp/terraform/commit/c00e929ee5f06e5e0265e24952ee55410e9fee26#diff-6665bbab4d1bce80874ff6467cd7bb85a08c9faaeaf8d2da121411b3a1ddbd55

My bad, the field is still lurking around in the codebase.

Reply:
@pkolyvas Do you want me to target that field for removal in 0.15?

Reply:
I pushed a commit adding that line back in.